src/shared/components/ShareWithColleagues/Modals/EditShareUserRightsModal.tsx

```diff
@@ -0,0 +1,108 @@
+import {
+	Button,
+	ButtonToolbar,
+	Form,
+	FormGroup,
+	Modal,
+	ModalBody,
+	ModalFooterRight,
+	Select,
+	Toolbar,
+	ToolbarItem,
+	ToolbarRight,
+} from '@viaa/avo2-components';
+import { capitalize } from 'lodash';
+import React, { FC, useEffect, useState } from 'react';
+
+import { tHtml, tText } from '../../../helpers/translate';
+import { ShareUserInfoRights } from '../ShareWithColleagues.types';
+
+type EditShareUserRightsModalProps = {
+	isOpen: boolean;
+	handleClose: () => void;
+	handleConfirm: (right: ShareUserInfoRights) => void;
+	currentRight: ShareUserInfoRights;
+};
+
+const EditShareUserRightsModal: FC<EditShareUserRightsModalProps> = ({
+	isOpen,
+	handleClose,
+	handleConfirm,
```

Comment:
Personal preference: 
for component parameters i usually use: onClose, onConfirm
for internal methods that need to handle some work before calling the onClose/onConfirm i use: handleClose, handleConfirm

But that is just personal preference.